db/migrate/20220509150139_create_reservation.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReservation < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reservations do |t|
+      t.references :user_id, null: false, foreign_key: true
```

Comment:
Active Record adds the "_id" for you. So, for it to work, just rename the column here to :user , like this

`t.references :user, null: false, foreign_key: true`
![Screen Shot 2022-05-09 at 10 37 33](https://user-images.githubusercontent.com/55037686/167446519-51e42da2-5185-465d-92c2-53dab73c8a30.png)